### DISCUSSION TITLE
[Chapter 04] 프론트 컨트롤러 패턴이란?

### DISCUSSION THREAD

**Discussion by JoisFe** [cowaine/spring-boot-for-msa#21] [Chapter 04. Spring Web MVC Overview]
DispatcherServlet은 프론트 컨트롤러 패턴으로 디자인 되어있다고 하였는데, 프론트 컨트롤러 패턴이 무엇인가요?

**Comment by corock** [comment]
우아한테크코스에 [10분 테코톡](https://youtu.be/calGCwG_B4Y) 동영상 중 프론트 컨트롤러 패턴에 대해서 잘 정리된 것이 있어 공유합니다.

_P.S._ 개인적인 여담으로, 특정 기술을 얘기할 때 뭐만 하면 XXX 패턴으로 명명해서 패턴이라는 용어에 조금 거부감이 들긴 합니다...
